### DISCUSSION TITLE
关于父目录的安全问题

### DISCUSSION THREAD

**Discussion by ErDongChen1973** [AlistGo/alist#685] [Ideas]
当我以管理员权限登录，然后复制某个文件的链接(不是下载直连)到其他浏览器时，不仅能看到分享的文件，还能进入到所有的父目录，并看到所有的内容(除非在元信息里面一级一级的添加密码)，感觉很不安全。请问能否一键设置禁止游客查阅父目录呢？

**Comment by xhofe** [comment]
- 根目录设置密码
- 设置-后端-打开检查父文件夹密码
- 要分享的文件夹单独设置密码（或者为空）

**Reply by ErDongChen1973** [reply]
你好！谢谢你的解答。这种方法我已经用过了，但除非每一级目录都设置密码，否则就算设置了上一级，上上级还是可以进去的。这对多级目录不太友好，能否给个设置所有父目录的方式呢？

**Reply by xhofe** [reply]
我不是说了吗 对根目录设置，打开`检查父文件夹密码`，对所有目录都会生效。

**Reply by ErDongChen1973** [reply]
谢谢！是我的操作问题! 不好意思，打扰了！

**Reply by deyangchu** [reply]
你好，V3中设置项里没有“后端”入口了， 请问还有办法设置吗？

**Reply by anwen-anyi** [reply]
> 你好，V3中设置项里没有“后端”入口了， 请问还有办法设置吗？

V3 自己在元信息添加然后设置密码勾选应用到子文件 不谢

**Reply by alimalls** [reply]
> > 你好，V3中设置项里没有“后端”入口了， 请问还有办法设置吗？
> 
> V3 自己在元信息添加然后设置密码勾选应用到子文件 不谢

大佬V3测试了，把应用到子件夹后，进入子文件夹不需要密码，是怎么什么原因。只有父文件夹生效，里面的子文件夹不起作用。

**Reply by anwen-anyi** [reply]
> > > 你好，V3中设置项里没有“后端”入口了， 请问还有办法设置吗？
> > 
> > 
> > V3 自己在元信息添加然后设置密码勾选应用到子文件 不谢
> 
> 大佬V3测试了，把应用到子件夹后，进入子文件夹不需要密码，是怎么什么原因。只有父文件夹生效，里面的子文件夹不起作用。

盲猜你元信息设置了两个不同的，其中不生效的子文件夹没有设置密码导致的

**Reply by alimalls** [reply]
> > > > 你好，V3中设置项里没有“后端”入口了， 请问还有办法设置吗？
> > > 
> > > 
> > > V3 自己在元信息添加然后设置密码勾选应用到子文件 不谢
> > 
> > 
> > 大佬V3测试了，把应用到子件夹后，进入子文件夹不需要密码，是怎么什么原因。只有父文件夹生效，里面的子文件夹不起作用。
> 
> 盲猜你元信息设置了两个不同的，其中不生效的子文件夹没有设置密码导致的

没有的，我只设置了一个父级文件夹【设计软件】，里面的子文件夹不起作用。
[https://ldbbs.ldmnq.com/bbs/topic/attachment/2023-3/ed5988b4-a1e2-4f8e-9601-fb40662e00b3.png](url)
[https://ldbbs.ldmnq.com/bbs/topic/attachment/2023-3/ed5988b4-a1e2-4f8e-9601-fb40662e00b3.png](url)
